packages/core/package.json

```diff
@@ -40,5 +40,11 @@
         "dist/**/*",
         "package.json",
         "README.md"
-    ]
+    ],
+    "bugs": { "url" : "https://github.com/doug-wade/tybalt/issues" },
+    "homepage": "https://tybalt.org",
```

Comment:
I think we can link to the page specific to each public package's page: https://tybalt.org/pages/core/